lib/whitehall_importer/integrity_checker.rb

```diff
@@ -87,6 +88,52 @@ def organisation_problems
       problems
     end
 
+    def unpublishing_problems
+      problems = []
+      return problems unless edition.withdrawn? || edition.removed?
+
+      unless expected_state?
+        problems << problem_description(
+          "edition state isn't expected",
+          "unpublished",
+          publishing_api_content.dig("publication_state"),
+        )
+        return problems
+      end
+
+      check = UnpublishingCheck.new(edition, publishing_api_unpublishing)
+
+      unless check.expected_type?
+        problems << problem_description(
+          "unpublishing type not expected",
+          publishing_api_content.dig("unpublishing", "type"),
+          check.expected_type,
+        )
+      end
+
+      unless check.expected_alternative_path?
+        problems << problem_description(
+          "unpublishing alternative path doesn't match",
+          publishing_api_unpublishing["alternative_path"],
+          edition.status.details.alternative_url,
```

Comment:
Might be good to have this as a method on check too for consistency with the other unpublishing checks

Reply:
I'd added a `#expected_alternative_path` method to the Unpublishing Check class.